client/rpcserver/rpcserver_test.go

```diff
@@ -130,11 +125,6 @@ func (c *TConn) SetWriteDeadline(_ time.Time) error {
 }
 
 func (c *TConn) Close() error {
-	// If the test has a non-nil close channel, signal close.
-	select {
-	case c.close <- struct{}{}:
-	default:
-	}
```

Comment:
Oh, this was the fix.  Did you still want to remove it for some reason?

Reply:
It's back.